kalite/topic_tools/annotate.py

```diff
@@ -61,8 +61,7 @@ def update_content_availability(content_list, language="en", channel="khan"):
 
             # Databases have been pre-filtered to only contain existing exercises
             # Assume if the assessment items have been downloaded, then everything is hunky dory.
-            if os.path.exists(contentload_settings.KHAN_ASSESSMENT_ITEM_VERSION_PATH):
-                update["available"] = True
+            update["available"] = True
```

Comment:
This change slowed down `annotate_content_items` quite a bit. From around 10s to 1 minute. 

This might be because we're now writing to the DB for every exercise. We should optimize this by marking all exercises as available in one query.